app/content/views/news.py

```diff
@@ -20,6 +21,44 @@ def get_serializer_class(self):
             return SimpleNewsSerializer
         return super().get_serializer_class()
 
+    def create(self, request, *args, **kwargs):
+        data = request.data
+        emojis_allowed = data.get("allowEmojis")
+        data.pop("allowEmojis", None)
+
+        serializer = NewsSerializer(data=data, context={"request": request})
+        try:
+            if serializer.is_valid():
+                news = super().perform_create(serializer)
+                if emojis_allowed:
+                    if not self.allow_emojis(
+                        data={"news": news.id, "emojis_allowed": emojis_allowed}
+                    ):
+                        return Response(
```

Comment:
Slå sammen de to if-testene for å unngå unødvendig nesting.

ved å først skrive emojies_allowed, og deretter not self.allow_emojies(...) du kan bruke emojies_allowed siden Python vil gå ut av testen før dette skjer hvis emojies_allowed er False.